src/daemon/gateway.go

```diff
@@ -707,3 +708,23 @@ func (gw *Gateway) GetAddressCount() (uint64, error) {
 
 	return uint64(len(allAccounts)), nil
 }
+
+func (gw *Gateway) GetNewSeed(entropyBits int) (seed *wallet.Seed, err error) {
```

Comment:
This does not need to be in gateway since it does not use networking, blockchain or wallets

It can do all of this logic in the http endpoint handler